pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -302,6 +305,14 @@ type CloudConfig struct {
 		//local VPC subnet (so load balancers can access it). E.g. 10.82.0.0/16 30000-32000.
 		DisableSecurityGroupIngress bool
 
+		//This flag enables the automatic shared security group ingress creation.
+		//Please check below to see the full mappings from flags to behavior
+		EnableSharedSecurityGroupIngress bool
```

Comment:
Many configurations don't use config, so we will probably need another way to enable this.